PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/serviceplan/ServicePlanComboBox.java

```diff
@@ -55,8 +55,6 @@ public void setSubscription(Subscription subscription) {
             this.clear();
             return;
         }
-        // force refresh service plan when switch subscription
-        Azure.az(AzureAppService.class).subscription(subscription.getId()).appServicePlans(true);
         this.refreshItems();
```

Comment:
you may just wrap these 2 lines with `runOnPooledThread`

Reply:
Changed to clear service plan cache instead of always force refresh.

For force refresh issue, now we usually add event listener to force refresh the explorer after resource creation(webapp/function), however, we didn't apply same framework to service plan, and I think we may need to do the cache refresh job in library rather than the toolkit